src/interface/api.yaml

```diff
@@ -2731,7 +2699,7 @@ components:
           description: Information about the Payee in the proposed financial transaction.
         payer:
           allOf:
-            - $ref: '#/components/schemas/PartyIdInfoTPLink'
+            - $ref: '#/components/schemas/PartyIdInfo'
```

Comment:
if we drop `PartyIdInfoTPLink` in favor of `PartyIdInfo` we will lost the validation which only allows `THIRD_PARTY_LINK` value

Reply:
Yep it's a temporary work around for a issue Lewis is having I'm just implementing the updates he needs.
The conversation is found here https://github.com/mojaloop/project/issues/2316